README.md

```diff
@@ -5,10 +5,64 @@
 
 ## Installation
 
+
 ```
 yarn add --dev jest-styled-components
 ```
 
+This package must be used with either `react-test-renderer` or `enzyme` and `enzyme-to-json` to generate a serialized `tree` from your component.
```

Comment:
This should be more generic, like "To render React components for testing you can use either [...]" because it doesn't apply to this package only.

Also, `enzyme-to-json` is only needed for snapshots testing.